src/triangulum/utils.clj

```diff
@@ -63,9 +63,20 @@
 
 (def ^:private path-env (System/getenv "PATH"))
 
+(defn sh-exec-with
+  "Provides a given path and environment to one bash command,
+  and parses the output, creating an array as described in `parse-as-sh-cmd`.
+  Returns a map in the following format: {:exit 0 :out 'Output message\n' :err ''}"
```

Comment:
Can you wrap the output in backticks (`)?

Reply:
Do you mean:
```suggestion
  Returns a map in the following format: {:exit 0 :out `Output message\n` :err ''}"
```

Reply:
Sorry, to be specific: 
```clojure
Returns a map in the following format: `{:exit 0 :out 'Output message'\n :err ''}"
```